website/docs/r/log_analytics_datasource_windows_event.html.markdown

```diff
@@ -0,0 +1,83 @@
+---
+subcategory: "Log Analytics"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_log_analytics_datasource_windows_event"
+description: |-
+  Manages a Log Analytics Windows Event DataSource.
+---
+
+# azurerm_log_analytics_datasource_windows_event
+
+Manages a Log Analytics Windows Event DataSource.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_log_analytics_workspace" "example" {
+  name                = "example-law"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  sku                 = "PerGB2018"
+}
+
+resource "azurerm_log_analytics_datasource_windows_performance_counter" "example" {
```

Comment:
Do we need to have a performance counter resource in the docs for the windows event resource?

Reply:
OMG, sorry about that!